Automattic/Gravatar-SDK-Android
# QuickEditor: Make ColorScheme configurable via GravatarColorScheme param

Closes #473 

### Description

This PR adds an additional param `GravatarColorScheme` that allows forcing Light or Dark color schemes within the Quick Editor. 

The passed param is used to set a proper `Configuration.uiMode` that's provided via `CompositionLocalProvider`. 
Thanks @hamorillo for the tip.


https://github.com/user-attachments/assets/f1672fe3-88aa-443f-b9c7-13a402980f5d


### Testing Steps

1. Start the demo app
2. Select a different ColorScheme than is currently set system-wise
3. Launch the QE
4. Confirm the color scheme is correct
5. Repeat with different color scheme
